backend/src/routes/api/service/modelregistry/index.ts

```diff
@@ -0,0 +1,34 @@
+import httpProxy from '@fastify/http-proxy';
+import { KubeFastifyInstance } from '../../../../types';
+import { DEV_MODE } from '../../../../utils/constants';
+import { getParam, setParam } from '../../../../utils/proxy';
+
+export default async (fastify: KubeFastifyInstance): Promise<void> => {
+  if (DEV_MODE) {
```

Comment:
This is not needed if your endpoint has OAuth.

Reply:
Nvm, Lucas notes there are still items going on here... they want it disabled for prod builds.